app/frontend/src/components/Avatar/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Avatar'
```

Comment:
Removal of the default export broke `SearchResult.tsx`.

```suggestion
export * from './Avatar';
export default Avatar;
```

Could also change `SearchResult.tsx`, but I imaging components will usually have a default export.